includes/Modules/Search_Console.php

```diff
@@ -276,8 +276,15 @@ protected function parse_data_response( Data_Request $data, $response ) {
 			case 'GET:matched-sites':
 				/* @var Google_Service_SearchConsole_SitesListResponse $response Response object. */
 				$entries = $this->map_sites( (array) $response->getSiteEntry() );
+				$strict  = filter_var( $data['strict'], FILTER_VALIDATE_BOOL );
```

Comment:
This is an invalid/undefined constant which will break the response if error reporting is strict.

The proper constant is [`FILTER_VALIDATE_BOOLEAN`](https://www.php.net/manual/en/filter.filters.validate.php)

Can you fix this @eugene-manuilov ?